app/controllers/admin/auctions/needs_attention_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::Auctions::NeedsAttentionController < Admin::BaseController
+  def index
+    @view_model = Admin::NeedsAttentionAuctionsViewModel.new
```

Comment:
No problemo

On Tuesday, August 2, 2016, Jessie A. Young <notifications@github.com>
wrote:

> In app/controllers/admin/auctions/needs_attention_controller.rb
> <https://github.com/18F/micropurchase/pull/987#discussion_r73193822>:
>
> > @@ -0,0 +1,5 @@
> > +class Admin::Auctions::NeedsAttentionController < Admin::BaseController
> > +  def index
> > +    @view_model = Admin::NeedsAttentionAuctionsViewModel.new
>
> yeah I did not do that for the other controller but probably makes sense.
> If you do it, mind also renaming
> https://github.com/18F/micropurchase/blob/develop/app/view_models/admin/closed_auctions_list_item_view_model.rb
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/micropurchase/pull/987/files/358f90120da907053c6e0f39e7bb1291c42e3871#r73193822>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFTBq1aEURip_BsNYXiyMvoot6JZj8Yks5qb3ZNgaJpZM4JatzG>
> .
>